app/views/songs/edit.html.erb

```diff
@@ -1,2 +1,7 @@
-<h1>Songs#edit</h1>
-<p>Find me in app/views/songs/edit.html.erb</p>
+<%= stylesheet_link_tag 'songs_manifest' %>
+<div class="form-page container">
+  <h1>Editing <%= @song.name %></h1>
+  <hr>
+
+  <%= render "form" %>
```

Comment:
One problem is that the button on the edit page says `Upload` when we are not uploading a new song. Two possible options:
1. Change the text of the button to Submit.
2. Take the button out of the form so we can customize it.

I think I prefer 1 but up to you.